example/main.tf

```diff
@@ -0,0 +1,235 @@
+terraform {
+  required_providers {
+    athenz = {
+      source = "yahoo/provider/athenz"
+    }
+  }
+}
+
+#terraform {
+#  required_providers {
+#    athenz = {
+#      source = "AthenZ/athenz"
+#      version = "1.0.31"
+#    }
+#  }
+#}
+
+locals {
+  user_name = "mshneorson"
+}
+
+provider "athenz" {
+  zms_url = "https://dev.zms.athens.yahoo.com:4443/zms/v1"
+  #  zms_url = "https://stage.zms.athenz.ouroath.com:4443/zms/v1"
+  cert = "/Users/${local.user_name}/.athenz/cert" # change to your location
+  key  = "/Users/${local.user_name}/.athenz/key"  # change to your location
+}
+
+module "mendi_msd_policies" {
+  source = "../msd_module/msd_policy"
```

Comment:
finally, how do use this module? without the source part

Reply:
no, with source part (and version). the source will be the path which we decide to push the moudle